stylesheets/styles.css

```diff
@@ -0,0 +1,290 @@
+
+.navigation-bar {
+  display: grid;
+  grid-template: 1fr / 1fr 3fr 1fr 1fr;
+}
+
+.navigation-bar li:nth-child(1) {
+  grid-column: 1/2;
+}
+
+.navigation-bar li:nth-child(2) {
+  grid-column: 3/4;
+}
+
+.navigation-bar li:nth-child(3) {
+  grid-column: 4/5;
+}
+
+li {
+  list-style-type: none;
+  font-size: 1rem;
+  color: indigo;
+  font-weight: bold;
+}
+
+nav li:hover {
+  font-size: 1.2rem;
+  color: teal;
+}
+
+h1, h2 {
+  font-family: 'Allerta Stencil', sans-serif;
+  color: indigo;
+  font-size: 4rem;
+}
+
+h2 {
+  font-size: 2rem;
+}
+
+.headline {
+  display: grid;
+  grid-template: 3fr 1fr 1.5fr / 100%;
+  max-height: 300px;
+}
+
+.headline h1 {
+  grid-row: 1 2;
+  font-family: 'Allerta Stencil', sans-serif;
+  color: indigo;
+}
+
+.headline sub {
+  grid-row: 2 3;
+  font-size: 1.75rem;
+}
+
+.headline a {
+  grid-row: 3 4;
+}
+
+.quote {
+  display: grid;
+  grid-template: 20% 40% 20% / 0.5fr 0.5fr 1fr 1fr 1fr 1fr 1fr;
+  font-style: italic;
+  color: indigo;
+}
+
+.quote blockquote {
+  grid-row: 2/3;
+  grid-column: 3/7;
+  font-size: 1rem;
+  line-height: 2rem;
+  text-align: left;
+}
+
+.quote p {
+  grid-row: 3/4;
+  grid-column: 6/8;
+}
+
+.why {
+  display: grid;
+  grid-template: 35% 65% / 1fr;
+}
+
+.why h2 {
+  font-size: 1rem;
+  grid-row: 1/2;
+  grid-column: 1/2;
+}
+
+.why ul {
+  grid-row: 2/3;
+  grid-column: 1/2;
+}
+
+.why ul {
+  display: grid;
+  grid-template: 100% / 20% 20% 20% 20% 20% ;
+
+}
+
+.why ul li:nth-of-type(1) {
+  grid-row: 1/2;
+  grid-column: 2/3;
+}
+
+.why ul li:nth-of-type(1) {
+  display: grid;
+  grid-template: 100% / 50% 50%;
+}
+
+.why ul li:nth-of-type(1) img {
+  grid-row: 1/2;
+  grid-column: 1/2;
+}
+
+.why ul li:nth-of-type(1) strong {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  font-size: 0.5rem;
+  text-align: left;
+}
+
+.why ul li:nth-of-type(1) p {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  font-size: 0.2rem;
+  text-align: left;
+}
+
+.why ul li:nth-of-type(2) {
+  grid-row: 1/2;
+  grid-column: 3/4;
+}
+
+.why ul li:nth-of-type(2) {
+  display: grid;
+  grid-template: 100% / 50% 50%;
+}
+
+.why ul li:nth-of-type(2) img {
+  grid-row: 1/2;
+  grid-column: 1/2;
+}
+
+.why ul li:nth-of-type(2) strong {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  font-size: 0.5rem;
+  text-align: left;
+
+}
+
+.why ul li:nth-of-type(2) p {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  font-size: 0.2rem;
+  text-align: left;
+}
+
+
+.why ul li:nth-of-type(3) {
+  grid-row: 1/2;
+  grid-column: 4/5;
+}
+
+.why ul li:nth-of-type(3) {
+  display: grid;
+  grid-template: 100% / 50% 50%;
+}
+
+.why ul li:nth-of-type(3) img {
+  grid-row: 1/2;
+  grid-column: 1/2;
+}
+
+.why ul li:nth-of-type(3) strong {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  font-size: 0.5rem;
+  text-align: left;
+
+}
+
+.why ul li:nth-of-type(3) p {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  font-size: 0.2rem;
+  text-align: left;
+}
+
+.meet {
+  display: grid;
+  grid-template: 10% 90% /23% 50% 27%;
+  min-height: 400px;
+}
+
+.meet h2 {
```

Comment:
I like the way you used row and column settings to control profile layout of team members